deploy/kubernetes/kube-prometheus-sumo-logic-mixin.libsonnet

```diff
@@ -4,149 +4,7 @@
     clusterName: "kubernetes"
   },
   sumologicCollector:: {
-    remoteWriteConfigs+: [
```

Comment:
Why did this file change? Something in the CI?

Reply:
weird, good catch Frank. @vsinghal13 can you take a look please?

Reply:
hmm.. looks like something in the CI. will take a look.

Reply:
The `build.sh` copies the remote write section from the `prometheus-overrides.yaml` to this file and then does a `git diff` to generate new files. But there is no change in the remote write config in this PR.